zipline/finance/commission.py

```diff
@@ -69,6 +69,18 @@ def calculate(self, order, transaction):
         """
         raise NotImplementedError('calculate')
 
+    def __eq__(self, other):
+        return self.asdict() == other.asdict()
+
+    def __hash__(self):
+        return hash((
+            type(self),
+            tuple(sorted(iteritems(self.asdict())))
```

Comment:
Is it important that these be hashable? I ask only because this implementation implies hash-ability of all the instance's attributes, which could be restrictive for subclass writers.